webservices/legal_docs/archived_murs.py

```diff
@@ -0,0 +1,398 @@
+        citation as cite
+    FROM MUR_ARCH.ARCHIVED_MURS
+    WHERE mur_id = %s AND citation is not null
+    ORDER BY citation
+"""
+
+MUR_SUBJECT_LV1 = """
+    SELECT
+        mur,
+        level_1,
+        subject
+    FROM MUR_ARCH.SUBJCT_FROM_FILE
+    WHERE level_2::integer = 0 AND level_3::integer = 0
+    AND mur = %s
+"""
+
+MUR_SUBJECT_LV2 = """
+    SELECT
+        mur,
+        level_2,
+        subject
+    FROM MUR_ARCH.SUBJCT_FROM_FILE
+    WHERE level_2::integer > 0
+    AND level_3::integer = 0
+    AND mur = %s
+    AND level_1 = %s
+"""
+
+MUR_SUBJECT_LV3 = """
+    SELECT
+        mur,
+        level_3,
+        subject
+    FROM MUR_ARCH.SUBJCT_FROM_FILE
+    WHERE level_3::integer > 0
+    AND mur = %s
+    AND level_1 = %s
+    AND level_2 = %s
+"""
+
+MUR_DOCUMENTS = """
+    SELECT
+        document_id,
+        mur_no,
+        pdf_text,
+        mur_id,
+        length,
+        url
+    FROM MUR_ARCH.DOCUMENTS
+    WHERE mur_id = %s
+    ORDER BY document_id
+"""
+
+INSERT_DOCUMENT = """
+    INSERT INTO MUR_ARCH.DOCUMENTS(
+        document_id,
+        mur_no,
+        pdf_text,
+        mur_id,
+        length,
+        url
+    ) VALUES (:document_id, :mur_no, :pdf_text, :mur_id, :length, :url)
+"""
+
+
+# To display the open_date and close_date of JSON format inside object "mur"
+class DateTimeEncoder(JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, (datetime.date, datetime.datetime)):
+            return obj.isoformat()
+
+
+def load_archived_murs(mur_no=None):
+
+    """
+    Reads data for Archived MURs from a Postgres database (under schema:mur_arch),
+    assembles a JSON document corresponding to the mur, and indexes this document
+    in Elasticsearch in the index `archived_murs` with a doc_type of `murs`.
+    """
+    es = utils.get_elasticsearch_connection()
+    logger.info("Loading archived mur{0}...".format(mur_no))
+    if LOCAL_DEBUG:
+        print("Loading archived mur{0}...".format(mur_no))
+
+    mur_count = 0
+    for mur in get_murs(mur_no):
+        if mur is not None:
+            logger.info("Loading archived murs: {0}".format(mur["no"]))
+            if LOCAL_DEBUG:
+                print("Loading archived murs: {0}".format(mur["no"]))
+
+            es.index("archived_murs", get_es_type(), mur, id=mur["doc_id"])
+            mur_count += 1
+
+            logger.info("{0} Archived Mur(s) loaded".format(mur_count))
+            if LOCAL_DEBUG:
+                print("{0} Archived Mur(s) loaded".format(mur_count))
+        else:
+            logger.info("Found an unpublished case - deleting archived mur: {0} from ES".format(mur["no"]))
+            es.delete_by_query(index="archived_murs", body={"query": {"term": {"no": mur["no"]}}},
+                doc_type=get_es_type())
+            logger.info("Successfully deleted archived mur(s) {} from ES".format(mur["no"]))
+
+        # ==for dubug use, display the JSON format of object "mur"
+        if LOCAL_DEBUG:
```

Comment:
can we use `logger.debug` here, or remove?

Reply:
good point. using logger.debug is good choice. Thanks